tasks/IKF.cpp

```diff
@@ -543,6 +548,11 @@ void IKF::writeOutput(IKFFilter & filter)
 	else
 	    orientation_out.angular_velocity = base::Vector3d::Zero();
         _orientation_samples_out.write(orientation_out);
+
+        acceleration_out.time = prev_ts;
+        acceleration_out.acceleration = filter.getAttitude().toRotationMatrix()*(acc_body - filter.getAccBias() - filter.getGravityinBody()); // world frame, since RigidBodyAcceleration is defined this way
```

Comment:
Since the current orientation of the filter is body in world, should it not be the inverse here?